app/assets/iframe/flex.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WorldPay DDC Flex 3DS</title>
+  </head>
+  <body>
+    <form id="collectionForm" name="devicedata" method="POST" action="https://secure-test.worldpay.com/shopper/3ds/ddc.html">
```

Comment:
There’s a different URL for live Worldpay accounts. Ideally, this should read from the environment variable (maybe the page needs to be dynamic?).

Reply:
Oops that‘s not meant to be there, it actually gets replaced by the JS anyway so the right one is used. But i had it there from early tests. Will remove

Reply:
Oops that‘s not meant to be there, it actually gets replaced by the JS anyway so the right one is used. But i had it there from early tests. Will remove